strictdoc/backend/excel/import_/excel_sheet_proxy.py

```diff
@@ -6,6 +7,11 @@
 from strictdoc.helpers.auto_described import auto_described
 
 
+class LIB(Enum):
```

Comment:
[Python convention](https://peps.python.org/pep-0008/#class-names): Class names should be CapWords. `Lib` not `LIB`

Reply:
There is also [precedent](https://github.com/strictdoc-project/strictdoc/blob/966eafbd977317c225f21fb9bffc7fac1e804da5/strictdoc/backend/sdoc_source_code/models/function_range_marker.py#L9) in this repo.